libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -558,7 +559,7 @@ - (void)beginUserAgentFlow {
     }
     
     // JWT Flow
-    if (self.credentials.jwt && self.credentials.instanceUrl) {
+    if (self.credentials.jwt) {
```

Comment:
This would have otherwise changed to checking the `nil`ness of `domain`, but we're checking it at the start, so no need.